### DISCUSSION TITLE
lottie-spm fails builds in GitHub Actions

### DISCUSSION THREAD

**Discussion by travismmorgan** [airbnb/lottie-ios#2328] [General]
Please remove linux error from lottie-spm/Package.swift to allow GitHub Actions to succeed.

`#if os(Linux)
#error("Linux is currently not supported")
#endif`

**Comment by calda** [comment]
What sort of job are you trying to run on Linux? Can you run the job on macOS instead? Could you ifdef out the lottie-spm dependency on Linux?

**Comment by rswave** [comment]
github dependabot fails on this

**Comment by hispanico94** [comment]
I encountered the same issue on Dependabot: https://github.com/airbnb/lottie-ios/discussions/2405
Like I said in the other discussion it seems that is not possible to fully disable Dependabot checks on a dependency. I'm wondering if that conditional inside the package manifest is still needed

**Comment by calda** [comment]
Thanks -- we removed this `#error` from our Package.swift: https://github.com/airbnb/lottie-spm/pull/27

**Reply by hispanico94** [reply]
Thank you for the fix! Do you know when it will be released?

**Reply by calda** [reply]
Just released 4.5.0 with this fix: https://github.com/airbnb/lottie-spm/releases/tag/4.5.0
